docs/source/_toctree.yml

```
@@ -262,6 +262,8 @@
       title: Splinter
     - local: model_doc/squeezebert
       title: SqueezeBERT
+    - local: model_doc/swin
+      title: Swin
```

```suggestion
      title: Swin Transformer
```
I guess the model is called "Swin Transformer" by the authors, right?